clientv3/retry.go

```diff
@@ -64,6 +82,14 @@ func RetryKVClient(c *Client) pb.KVClient {
 	return &retryKVClient{pb.NewKVClient(c.conn), c.retryWrapper}
 }
 
+func (rkv *retryKVClient) Range(ctx context.Context, in *pb.RangeRequest, opts ...grpc.CallOption) (resp *pb.RangeResponse, err error) {
```

Comment:
Range shouldn't use retryKVClient; it's meant for safely retrying mutable operations.